### PR TITLE
Feat/create buy and sell key functions

### PR DESCRIPTION
**Pull request by roy105703007**

### CODE REVIEW

File: script/DeployCreatorTech.s.sol [hunk 1]
```diff
@@ -0,0 +1,23 @@
+// SPDX-License-Identifier: UNLICENSED
+pragma solidity 0.8.22;
+
+import {Script, console2} from "forge-std/Script.sol";
+import {CreatorTech} from "../src/CreatorTech.sol";
+
+contract DeployScript is Script {
```
Comment (jimmyx0x) [inline script/DeployCreatorTech.s.sol:7]: In reality, Hologram will probably have a single signer key. Perhaps we can have this be an input argument in the run() function below

File: test/CTBindCreator.t.sol [hunk 2]
```diff
@@ -42,12 +36,20 @@ contract CreatorTechTest is Test {
         assertEq(ctCreatorAddr, creatorAddr);
     }
 
+    function testBindCreatorAndClaim_haveUnclaimedFees() public {}
```
Comment (jimmyx0x) [inline test/CTBindCreator.t.sol:39]: nit: What's the plan with these? Perhaps can write some TODOs if that's the plan.

**Review by jimmyx0x — APPROVED** [review]
Left some nits

File: script/DeployCreatorTech.s.sol (hunk 1 above)
Comment (roy105703007) [reply script/DeployCreatorTech.s.sol:7]: done

File: test/CTBindCreator.t.sol (hunk 2 above)
Comment (roy105703007) [reply test/CTBindCreator.t.sol]: fill this

**Pull request merged**